addons/field_rations/stringtable.xml

```diff
@@ -630,13 +659,15 @@
         </Key>
         <Key ID="STR_ACEX_Field_Rations_MRE_BeefStew_DisplayName">
             <English>MRE Beef Stew</English>
+            <German>EPA Rindereintopf</German>
             <Chinese>軍糧 牛肉燉湯</Chinese>
             <Japanese>MRE ビーフ シチュー</Japanese>
             <Polish>MRE Gulasz wołowy</Polish>
             <Russian>ИРП: Жаркое из говядины</Russian>
         </Key>
         <Key ID="STR_ACEX_Field_Rations_MRE_BeefStew_Description">
             <English>An MRE containing Beef Stew. Heat for best effect</English>
+            <German>Eine EPA welche Rindereintopf beeinhaltet. Erwärme für den besten Effekt</German>
```

Comment:
, welche

Reply:
> Erwärme für den besten Effekt

Don't quite like this.
Maybe "Für den besten Effekt warm genießen" or smth like that